src/set_theory/cardinal.lean

```diff
@@ -74,6 +74,9 @@ theorem mk_le_of_injective {α β : Type u} {f : α → β} (hf : injective f) :
 theorem mk_le_of_surjective {α β : Type u} {f : α → β} (hf : surjective f) : mk β ≤ mk α :=
 ⟨embedding.of_surjective hf⟩
 
+theorem mk_le_of_subtype {α : Type u} (p : α → Prop) : mk (subtype p) ≤ mk α :=
```

Comment:
This already basically exists as `mk_subtype_le`, but maybe it doesn't hurt to have a version with `subtype` instead of a coercion?

Reply:
How did I miss `mk_subtype_le`? I'd remove `mk_le_of_subtype` (why did I choose such a bad name?).

Reply:
We had both `mk_subtype_le` and `mk_set_le` with the same signature. Now `mk_set_le` takes a set, and `mk_subtype_le` takes a map `α → Prop`.